examples/client/lwm2mclient.c

```diff
@@ -95,6 +95,7 @@ lwm2m_object_t * objArray[OBJ_COUNT];
 # define BACKUP_OBJECT_COUNT 2
 lwm2m_object_t * backupObjectArray[BACKUP_OBJECT_COUNT];
 
+// Structure to hold client data
 typedef struct
 {
     lwm2m_object_t * securityObjP;
```

Comment:
e.g. this one has this definition,thus it should be referenced here:
https://github.com/eniqo-energy/wakaama/blob/9b09fac16a855bf9be908d6e0094b6611703daef/include/liblwm2m.h#L503